packages/manager/modules/nutanix/src/template.html

```diff
@@ -0,0 +1,89 @@
+<div>
+    <oui-header data-heading="{{ ::'nutanix_title' | translate }}">
+        <oui-guide-menu data-text="{{ ::'nutanix_guides' | translate }}">
+            <oui-guide-menu-item
+                data-href="{{:: $ctrl.guideUrl }}"
+                data-external
+                ><span data-translate="nutanix_guides_main"></span>
+            </oui-guide-menu-item>
+        </oui-guide-menu>
+    </oui-header>
+    <oui-datagrid rows="$ctrl.clusters" data-page-size="25" class="mt-5">
+        <oui-datagrid-topbar>
+            <oui-spinner data-ng-if="ctrl.delete.loading"></oui-spinner>
+        </oui-datagrid-topbar>
+        <oui-datagrid-column
+            title="'nutanix_cluster_list_name' | translate"
+            property="serviceName"
+            data-sortable
+            data-searchable
+            data-filterable
+        >
+            <a
+                class="oui-link"
+                ng-href="{{:: $ctrl.getClusterDetailsState($row.serviceName) }}"
+                data-ng-bind="$row.serviceName"
```

Comment:
i think we have to made both one time binding or not, because both had a relationship.
```suggestion
                ng-href="{{:: $ctrl.getClusterDetailsState($row.serviceName) }}"
                data-ng-bind=":: $row.serviceName"
```